fhir-server-test/src/test/java/com/ibm/fhir/server/test/DeleteTest.java

```diff
@@ -430,17 +435,31 @@ public void testConditionalDeleteResource() throws Exception {
         } else {
             assertResponse(response.getResponse(), Response.Status.METHOD_NOT_ALLOWED.getStatusCode());
         }
-        
-        // A search that results in multiple matches should result in a 412 status code.
+
+        // A search that results in multiple matches:
+        // (1) if matches > FHIRConstants.FHIR_CONDITIONAL_DELETE_MAX_NUMBER_DEFAULT, then result in a 400 status code.
```

Comment:
should it be a 412 to be more consistent with the fhir spec?

Reply:
yes, I'm Ok to change to use 412.

Reply:
done